lcd.lua

```diff
@@ -211,16 +229,21 @@ minetest.register_node("digilines:lcd", {
 	light_source = 6,
 })
 
+minetest.register_lbm({
+	label = "Replace Missing Text Entities",
+	name = "digilines:replace_text",
+	nodenames = {"digilines:lcd"},
+	run_at_every_load = true,
+	action = function(pos)
+		prepare_writing(pos)
```

Comment:
You could inline that. `action = prepare_writing`

Reply:
Fixed in other places too :+1: